cmd/policy.go

```diff
@@ -83,12 +84,27 @@ func (sys *PolicySys) Init(objAPI ObjectLayer) error {
 	for _, bucket := range buckets {
 		config, err := GetPolicyConfig(objAPI, bucket.Name)
 		if err != nil {
-			if _, ok := err.(BucketPolicyNotFound); !ok {
-				return err
+			if _, ok := err.(BucketPolicyNotFound); ok {
+				continue
```

Comment:
I don't think its necessary to change this `if` block i.e. migration is always happens in `else` block.